.gitmodules

```diff
@@ -55,3 +55,6 @@
 [submodule "src/rapidjson"]
 	path = src/rapidjson
 	url = https://github.com/ceph/rapidjson
+[submodule "src/benchmark"]
```

Comment:
instead of adding google/benchmark as a submodule, can we add it using `ExternalProject_Add()`? and do the git clone on the fly? just like how we add `nvml`.

Reply:
yeah, np. i'm not convinced that it's worth pulling in a new library for this dumb little benchmark though, unless we plan to make further use of it. do you think it's worth merging those pieces?

Reply:
yeah, makes sense. after a second thought, neither am i convinced now. it'd be helpful in the case that we have multiple comparable backend/implementations of a certain feature like lz4,snappy,zlib and zstd, and user/developer can use the benchmark tool to evaluate them with different datasets/parameters to choose a backend and a set of parameters. but apparently, `for_each_substr()` does not have its alternative(s) in Ceph, so probably we should drop this benchmark tool.